module-2/lab-df-calculation-and-transformation/your-code/ch1.ipynb

```diff
@@ -0,0 +1,1621 @@
+{
```

Comment:
- Try to not use as a variable the same name as a Python built-in function. It can cause unexpected behaviour. 
```Python
max=pokemon['A/D Ratio'].max()
min=pokemon['A/D Ratio'].min()
```
- Another way of getting the guessed total column: 
```Python
pokemon['Guessed Total'] = pokemon.loc[:,'HP':'Speed'].sum(axis = 1)
```
- Practicando lambda: an easier way could be the following 
```Python
pokemon['Guessed Total2']=pokemon.apply(lambda s: s['HP':'Speed'].sum(), axis=1)
pokemon.head(2)
```